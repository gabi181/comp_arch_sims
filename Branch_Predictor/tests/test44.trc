2 2 24 0 global_history local_tables not_using_share
0x4C53B578 N 0x0D6C79A4
0x4123F5F0 T 0x3BD70E9C
0x4123F5F0 N 0x3BD70E9C
0x193626C8 N 0x08FAF604
0x193626C8 T 0x08FAF604
0x4C53B578 N 0x0D6C79A4
0x4C53B578 N 0x0D6C79A4
0x4123F5F0 T 0x3BD70E9C
0x4C53B578 T 0x0D6C79A4
0x4123F5F0 T 0x3BD70E9C
0x193626C8 N 0x08FAF604
0x4C53B578 T 0x0D6C79A4
0x4123F5F0 N 0x3BD70E9C
0x4C53B578 T 0x0D6C79A4
0x193626C8 N 0x08FAF604
0x2F528184 T 0x69908100
0x193626C8 N 0x08FAF604
0x4C53B578 N 0x0D6C79A4
0x2F528184 T 0x69908100
0x2F528184 T 0x69908100
0x2F528184 T 0x69908100
0x2F528184 N 0x69908100
0x193626C8 T 0x08FAF604
0x193626C8 T 0x08FAF604
0x4C53B578 N 0x0D6C79A4
