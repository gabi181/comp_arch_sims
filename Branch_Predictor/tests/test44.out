0x4c53b578 N 0x4c53b57c
0x4123f5f0 N 0x4123f5f4
0x4123f5f0 N 0x4123f5f4
0x193626c8 N 0x193626cc
0x193626c8 N 0x193626cc
0x4c53b578 N 0x4c53b57c
0x4c53b578 N 0x4c53b57c
0x4123f5f0 N 0x4123f5f4
0x4c53b578 N 0x4c53b57c
0x4123f5f0 N 0x4123f5f4
0x193626c8 N 0x193626cc
0x4c53b578 N 0x4c53b57c
0x4123f5f0 N 0x4123f5f4
0x4c53b578 N 0x4c53b57c
0x193626c8 N 0x193626cc
0x2f528184 N 0x2f528188
0x193626c8 N 0x193626cc
0x4c53b578 N 0x4c53b57c
0x2f528184 N 0x2f528188
0x2f528184 N 0x2f528188
0x2f528184 N 0x2f528188
0x2f528184 N 0x2f528188
0x193626c8 N 0x193626cc
0x193626c8 N 0x193626cc
0x4c53b578 N 0x4c53b57c
flush_num: 13, br_num: 25, size: 128b
